src/models/properties.rs

```diff
@@ -284,9 +286,10 @@ pub enum PropertyValue {
         formula: FormulaResultValue,
     },
     /// <https://developers.notion.com/reference/page#relation-property-values>
+    /// It is actually an array of relations
     Relation {
         id: PropertyId,
-        relation: RelationValue,
+        relation: Option<Vec<RelationValue>>
```

Comment:
I generally leverage `#[serde(default)]` to have it be an empty `Vec` instead of having the `Option`. Is there a need to differentiate empty vs not existent? Thoughts @jakeswenson?

Reply:
Files entry below for example is 
```
    Files {
        id: PropertyId,
        files: Option<Vec<FileReference>>,
    },
```
So i used same style

Reply:
Yep, saw that! More of a general note.

Reply:
Good point, I suppose I prefer the empty vector too provided that it doesn't cause any issues differentiating between no relation and an empty relation. (Which I don't think it should here.)

Reply:
But we can address this consistently in another pass.